libpod/pod.go

```diff
@@ -68,8 +89,14 @@ type PodInspectState struct {
 
 // PodContainerInfo keeps information on a container in a pod
 type PodContainerInfo struct {
-	ID    string `json:"id"`
-	State string `json:"state"`
+	ID      string `json:"id"`
+	State   string `json:"state"`
+	IsPause bool   `json:"isPause,omitempty"`
```

Comment:
No need for this - this is exclusively for `Inspect` which will should print the ID of the pause container separately